### PR TITLE
update swift-transformers

### PR DESCRIPTION
**Pull request by maiqingqiang**
Update to swift-transformers #117

### CODE REVIEW

File: Package.resolved [hunk 1]
```diff
@@ -9,22 +9,31 @@
         "version" : "6.0.1"
       }
     },
+    {
+      "identity" : "jinja",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/maiqingqiang/Jinja",
+      "state" : {
+        "branch" : "main",
+        "revision" : "5b0703d19a8901b76948753e5c5e3ca77043d33f"
```
Comment (davidkoski) [inline Package.resolved:18]: Can you make a tag for this?  As people use mlx-swift-examples in their own code they had expressed a preference for tags.

File: Package.resolved [hunk 2]
```diff
@@ -59,8 +68,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/huggingface/swift-transformers",
       "state" : {
-        "revision" : "4f915610451d29a05948802a140880ff37494dad",
-        "version" : "0.1.6"
+        "branch" : "main",
+        "revision" : "5d89b5d9ab5a25d4337c42dbc559d391c966c912"
```
Comment (davidkoski) [inline Package.resolved:72]: Maybe we should wait for a tag here too.

**Comment by davidkoski** [comment]
Looks good -- let's wait for a tag on the swift-transformers.  I can update the xcodeproj when ready

**Comment by maiqingqiang** [comment]
@davidkoski I have already updated swift-transformers 0.1.11.

**Comment by davidkoski** [comment]
OK, I will take a look this afternoon!

**Review by davidkoski — APPROVED** [review]

**Pull request merged**